zerver/lib/upload/base.py

```diff
@@ -52,7 +51,7 @@ def sanitize_name(value: str) -> str:
     value = re.sub(r"[^\w\s.-]", "", value).strip()
     value = re.sub(r"[-\s]+", "-", value)
     assert value not in {"", ".", ".."}
-    return Markup(value)
+    return value
```

Comment:
What is the practical effect of removing `Markup` here? I don't really understand why that would have been appropriate in this location given the description of generating a value for a filename.

Do the comments on how this relates to `django.utils.text.slugify` need to change here?

Reply:
No practical effect, since the characters we allow are not special in HTML. There’s no reason for the `Markup` to have been here. It was removed from `django.utils.text.slugify` in 2.2a1 (django/django#10181).

Reply:
Cool, thanks for the explanation.